cdc/owner/changefeed.go

```diff
@@ -341,6 +345,13 @@ func (c *changefeed) tick(ctx cdcContext.Context, state *orchestrator.Changefeed
 				newResolvedTs = prevResolvedTs
 			}
 		}
+		// checkpointTs and resolvedTs should never regress.
+		if newResolvedTs < prevResolvedTs {
```

Comment:
when will this case happen?

Reply:
Whe the owner is just initialized. It loads DDL starts at `checkpointTs-1`, which means resolvedTs can regress.